Makefile

```diff
@@ -66,6 +69,9 @@ lint-backend:  ## Run linters on backend container
 lint-frontend:  ## Run linters on frontend container
 	docker-compose run --rm --no-deps frontend npm run lint:fix
 
+format-backend:  ## Run formatters on backend
```

Comment:
`format` vs `lint` - this is inconsistent, we should use one name for frontend and backend

Reply:
totally agree, what I wanted to do here is to split the commands into three categories - `lint`, `format`, `test`, where `lint-*` checks for code style with complex rules (`eslint`/`flake8`), `format-*` checks for unified code style (lower level than linter, `@vue/cli-plugin-eslint`/`black`) and `test-*`, well, tests (jest/pytest).

Though that might be bit too overcomplicated? I mean, I can merge the `format-backend` with `lint-backend` and leave it like that.

Reply:
I don't see when would we need separate `lint` and `format` - I would merge it.

Reply:
okay, will do